app/service/task_status_service.rb

```diff
@@ -60,6 +61,14 @@ def what_happens_next_status
     end
   end
 
+  def payment_link_status
+    if @form.payment_url.present?
+      :completed
+    else
+      :optional
+    end
+  end
+
```

Comment:
```suggestion
  def payment_link_status
    return :completed if @form.payment_url.present?
    
    :optional
  end

```

Reply:
I'm surprised Rubocop didn't suggest this...

Reply:
Bleurgh... because govuk disabled it https://github.com/alphagov/rubocop-govuk/blob/main/config/style.yml#L125

Reply:
Looks like it's the same throughout this file (not that that's a reason not to change it now)

Reply:
sorted!